lib/avromatic.rb

```diff
@@ -27,9 +27,15 @@ def self.configure
 
   def self.build_schema_registry
     raise 'Avromatic must be configured with a registry_url' unless registry_url
-    AvroTurf::CachedConfluentSchemaRegistry.new(
-      AvroTurf::ConfluentSchemaRegistry.new(registry_url, logger: logger)
-    )
+    if use_cacheable_schema_registration
```

Comment:
Is there a better name for this setting to disambiguate the two forms of caching (local vs. reverse proxy)? It's a bit confusing to see caching in both branches of the conditional.